web/docs/advanced/apis.md

```diff
@@ -136,48 +136,23 @@ For example, if your app is running at `https://example.com` then from the above
 
 To use the API from your client, including with auth support, you can import the Axios wrapper from `wasp/client/api` and invoke a call. For example:
 
-<Tabs groupId="js-ts">
-<TabItem value="js" label="JavaScript">
-
-```jsx title="src/pages/SomePage.jsx"
-import React, { useEffect } from "react";
-import { api } from "wasp/client/api";
+```tsx title="src/pages/SomePage.tsx" auto-js with-hole
+import React, { useEffect } from 'react'
+import { api } from 'wasp/client/api'
 
 async function fetchCustomRoute() {
-  const res = await api.get("/foo/bar");
-  console.log(res.data);
+  const res = await api.get('/foo/bar')
+  console.log(res.data)
 }
 
 export const Foo = () => {
   useEffect(() => {
-    fetchCustomRoute();
-  }, []);
-
-  return <>// ...</>;
```

Comment:
Hm I wonder what this `//` was doing here?

Reply:
Probably just a typo 🤷🏼‍♂️